model-optimizer/extensions/middle/EltwiseChecker.py

```diff
@@ -63,7 +63,7 @@ def mark_eltwise_node(self, node, feature_channel=None):
         feature_dim = min(1, tensor_shape.size - 1) if node.graph.graph['layout'] == 'NCHW' else -1
         if feature_channel is not None:
             feature_dim = feature_channel
-        ones = np.ones(len(tensor_shape))
+        ones = np.ones(len(tensor_shape), dtype=np.float32)
```

Comment:
this is all about shapes. why do we make them float? can we make helper for ones and zeros with the same logic as mo_array? mo_zeros, mo_ones. Also -- only in the code, not in the tests.

Reply:
Agreed to abandon the idea of mo_zeros and mo_ones but check that all the types are valid -- especially for the integer cases